style.css

```diff
@@ -102,3 +165,142 @@ main {
   align-self: center;
   margin-top: 2.5rem;
 }
+
+section {
+  margin-top: 4em;
+}
+
+.desc {
+  width: 327px;
+  height: 97px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  margin: 26px auto;
+  color: #172b4d;
+  padding: 1em;
+  margin-top: -0.5em;
+}
+
+.divider {
+  width: 48px;
+  height: 4px;
+  background-color: #36b37e;
+  border-radius: 24px;
+  margin: 80px auto 0;
+}
+
+.cards {
+  margin: 0 20px;
+  border-radius: 5px;
+  padding-bottom: 20px;
+  border: 2px solid #ebf0ee;
+}
+
+.card {
+  text-align: center;
+}
```

Comment:
- [ ] Kindly make sure the content of each card is aligned in each container using CSS flexbox in order to meet the project requirement. :wink: